src/sentry/models/groupinbox.py

```diff
@@ -29,3 +29,18 @@ class GroupInbox(Model):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_groupinbox"
+
+
+def add_group_to_inbox(group, reason, reason_details=None):
+    gi, created = GroupInbox.objects.get_or_create(
+        group=group, defaults={"reason": reason.value, "reason_details": reason_details}
+    )
+    return gi
+
+
+def remove_group_from_inbox(group):
+    try:
+        group = GroupInbox.objects.get(group=group)
+        group.delete()
```

Comment:
Its confusing that you reuse the name `group` for the inbox item when it was the Group.

Reply:
I agree, changed this.